src/main/java/com/maddisportfolio/user/UserService.java

```diff
@@ -96,6 +96,14 @@ else if(lastName != null && !lastName.equals("")){
                 acceptedSender.get().setFriendsWithLoggedInUser(true);
             }
         }
+        List<FriendRequest> declinedRequestToLoggedInUser = friendRequestDao.findAllByRecipientAndFriendRequestStatus(loggedInUser, FriendRequestStatus.DECLINED);
+        for(FriendRequest friendRequest : declinedRequestToLoggedInUser) {
+            Optional<User> correspondingSender = searchResults.stream().filter(x -> x.getId() == friendRequest.getSender().getId()).findFirst();
+            if (correspondingSender.isPresent()) {
+                loggedInUser.setHasReceivedRequestFromSearchedUser(false);
```

Comment:
I'm not sure this is doing anything. I'm looking at the search-results template and I'm not seeing anywhere where we're reading this value from the logged in user